app/controllers/private_api_controller.rb

```diff
@@ -73,6 +75,9 @@ def proteins
     # sort entries
     @entries = @entries.to_a.sort_by { |e| e.taxon.nil? ? '' : e.taxon.name }
 
+    # puts @entries.inspect
+    puts @lineages.inspect
```

Comment:
this shouldn't be here?

Reply:
Oops, forgot to remove these. They're gone now :)